cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -193,7 +205,7 @@ def _use_sample_sheet_from_housekeeper(self, flow_cell: FlowCellDirectoryData) -
                 f"Sample sheet for flow cell {flow_cell.id} does not exist in Housekeeper"
             )
         self.validate_sample_sheet(
-            sample_sheet_path=sample_sheet_path, bcl_converter=self.bcl_converter
+            sample_sheet_path=sample_sheet_path, bcl_converter=flow_cell.bcl_converter
```

Comment:
Also here, do we need the bcl_converter going forward? If it cannot be validated with bclconvert - it needs to be translated, right?

Reply:
It is still possible to validate a `bcl2fastq` sample sheet using `cg demultiplex samplesheet validate -b bcl2fastq` and demultiplex with `bcl2fastq` using `cg demultiplex flow-cell -b bcl2fastq`. These commands require this function to be able to validate a `bcl2fastq` sheet. Removing these functionalities is the next step and when this happens, we can remove this bcl_converter from here. Did I answer your question? @ChrOertlin